tools/getDepartures.php

```diff
@@ -0,0 +1,13 @@
+<form id="departuresForm">
+    <div class="input-group">
+        <button type="button" class="btn btn-primary" id="getLocation"><i class="fa-solid fa-location-crosshairs"></i> Get Location</button>
```

Comment:
a seperate input for manual location would be a great bonus, so if you use this with your pc you dont depend on IP Locations (which can be off)

Reply:
I will bring this in a future Update :)